test/lib/pipeline.test.js

```diff
@@ -639,6 +639,52 @@ describe('Pipeline Model', () => {
             });
         });
 
+        it('Only gets PR jobs', () => {
```

Comment:
Should we also add a test where if we don't specify a config type, it returns all jobs?

Reply:
I believe that test case is already handled previously in the [older tests](https://github.com/screwdriver-cd/models/blob/master/test/lib/pipeline.test.js#L601-L640) above this one where no config is given.

Reply:
mkay 👍